src/robot.coffee

```diff
@@ -393,26 +407,6 @@ class Robot
         scriptDocumentation.commands.push cleanedLine
         @commands.push cleanedLine
 
-  # Public: A helper send function which delegates to the adapter's send
-  # function.
-  #
-  # user    - A User instance.
-  # strings - One or more Strings for each message to send.
-  #
-  # Returns nothing.
-  send: (user, strings...) ->
-    @adapter.send user, strings...
```

Comment:
I don't think I've ever used this myself, but I did a google for 'hubot "robot.reply"' and see some references to it out in the wild. It might be better to deprecate it, if we want to avoid a major version bump.